### DISCUSSION TITLE
Hiding staff lines

### DISCUSSION THREAD

**Discussion by infojunkie** [w3c-cg/musicxml#559] [Q&A] [answered]
I am trying to achieve the following style:

![9 20-special-1](https://github.com/user-attachments/assets/3a3bbb80-8b2c-4c09-942f-0d81c6e6c2b3)

I am stuck on hiding the staff lines. My reference is Finale 27.4.x. When I try:
```xml
<attributes>
  <staff-details>
        <staff-lines>5</staff-lines>
        <line-detail line="1" print-object="no"/>
        <line-detail line="2" print-object="no"/>
        <line-detail line="3" print-object="no"/>
        <line-detail line="4" print-object="no"/>
        <line-detail line="5" print-object="no"/>
  </staff-details>
</attributes>
```
Finale shows the staff lines. When I try `<staff-details print-object="no">`  then Finale hides the whole staff, including the notes. 

Which one should achieve the desired effect?

**Comment by mscuthbert** [comment, marked as answer]
Change staff lines to 0 and delete all the staff-details objects.

**Comment by infojunkie** [comment]
Interesting idea, thanks! Finale ignored this instruction, unfortunately, and so did MuseScore 3 / 4. But I will use this convention in my own [MusicXML-to-MuseScore converter](https://github.com/infojunkie/musicxml-mscx) to hide the staff lines.
